### ISSUE TITLE
Custom build configurations - Aws Toolkit for VS 2017

### ISSUE THREAD

**Issue by pawepaw** [aws/aws-sdk-net#679]
Hi,

I'm using aws toolkit for VS 2017. I wanted to use more build configurations than debug/release but after adding new build configuration i've spotted that in publish to aws lambda popup I cannot chose my custom build configuration (only debug/release are available). 

Is it hardcoded to Debug/Release only? Shouldn't it be taken from solution build configurations?

**Comment by normj** [comment]
You are correct right now it is hard coded for debug and release because in the project.json days that was all that valid. I'll get that added to our backlog to look up the configurations from the csproj file.

**Comment by pawepaw** [comment]
Thanks for answer. Would be great to add it asap because it seems like cardinal funcionality.
